src/ocean/util/container/more/Stack.d

```diff
@@ -1,309 +1,316 @@
 /*******************************************************************************
```

Comment:
This module had exactly 0 tests so I had to do something to actually check that things work. Now that they do, we could possibly move all the part adding tests & style to minor but it would be a bit scary to merge fix without tests coming in a pair.

Reply:
The massive coding style changes are more what I was referring to.